src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Truck truck = new Truck();
```

Comment:
```suggestion
        Machine truck = new Truck();
```
as a reference type it's better to use more abstract type (interface or abstract class), that will simplify usage of polymorphism